src/extensions/mvs/tree/molstar/conversion.ts

```diff
@@ -46,6 +47,13 @@ const mvsToMolstarConversionRules: ConversionRules<FullMVSTree, MolstarTree> = {
             { kind: 'structure', params: omitObjectKeys(node.params, ['block_header', 'block_index', 'model_index']) },
         ] satisfies MolstarNode[];
     },
+    'raw_volume': (node, parent) => {
+        if (parent?.kind !== 'parse') throw new Error('Parent of "structure" must be "parse".');
+        // const { format } = ParseFormatMvsToMolstar[parent.params.format];
+        return [
+            { kind: 'raw_volume', params: { source: node.params.source, options: node.params.options } }
+        ] satisfies MolstarNode[];
+    }
```

Comment:
If a node is converted into exactly the same node, it shouldn't be included here. This is only for non-trivial conversion rules.

Reply:
Ok, got it, thanks. Removed it already.